src/Tree.tsx

```diff
@@ -64,7 +64,7 @@ interface CheckInfo<TreeDataType extends BasicDataNode = DataNode> {
 }
 
 export interface AllowDropOptions<TreeDataType extends BasicDataNode = DataNode> {
-  dragNode: TreeDataType;
+  // dragNode: TreeDataType;
```

Comment:
Should it be removed? Or we can change it to `dragNode?: TreeDataType`

Reply:
Sometimes it's important to know that dragNode to determine if it's able to drop.

Reply:
Yes it could be optional